### ISSUE TITLE
A feature that, instead of a wall hack, uses a 'sensor.'

### ISSUE THREAD

**Issue by TheChosenOne2325** [sailro/EscapeFromTarkov-Trainer#560]
It comes from this trainer: 'https://www.kronzky.info/eft/trainer.html'. Instead of the usual wall hack, it provides a direction indicating where the enemy might be while moving.

**Comment by sailro** [comment]
Kronzky's trainer is a fork of this one but I don't know if the source code is still available.

**Comment by TheChosenOne2325** [comment]
I think its a good feature for those who don't want to use a wallhack, personally i would love somethig similar

**Comment by sailro** [comment]
Yes why not, I'm going to have a look at the built dll, it is always possible to see how it's done

**Comment by JohnC0de** [comment]
I dont know if it helps, but theres a mod similar: https://hub.sp-tarkov.com/files/file/2395-audio-accessibility-indicators/

**Comment by sailro** [comment]
Really nice, thank you @JohnC0de.

Closing as it very similar to the initial request.
